### PR TITLE
Add notes to plantings in backend.

### PR DESCRIPTION
**Pull request by horenso**
## Basics

- [x] I added a line to [changelog.md](/doc/changelog.md)
- [x] The PR is rebased with current master.
- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages and changelog.
- [x] The buildserver is happy.

## Checklist

- [x] I have installed and I am using [pre-commit hooks](../doc/contrib/README.md#Hooks)
- [x] I fully described what my PR does in the documentation
- [x] I fixed all affected documentation
- [x] I fixed the introduction tour
- [x] I wrote migrations in a way that they are compatible with already present data
- [x] I fixed all affected decisions
- [x] I added automated tests or a [manual test protocol](../doc/tests/manual/protocol.md)
- [x] I added code comments, logging, and assertions as appropriate
- [x] I translated all strings visible to the user
- [x] I mentioned [every code or binary](https://github.com/ElektraInitiative/PermaplanT/blob/master/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)
- [x] I created left-over issues for things that are still to be done
- [x] Code is conforming to [our Architecture](/doc/architecture)
- [x] Code is conforming to [our Guidelines](/doc/guidelines)
- [x] Code is consistent to [our Design Decisions](/doc/decisions)
- [x] Exceptions to any guidelines are documented

## Review

- [x] I've tested the code
- [x] I've read through the whole code
- [x] I've read through the whole documentation
- [x] I've checked conformity to guidelines

### CODE REVIEW

File: backend/src/model/dto/plantings.rs [hunk 1]
```diff
@@ -46,6 +46,8 @@ pub struct PlantingDto {
     pub additional_name: Option<String>,
     /// Is the planting an area of plants.
     pub is_area: bool,
+    /// Notes about the planting in Markdown.
+    pub notes: Option<String>,
```
Comment (danielsteinkogler) [inline backend/src/model/dto/plantings.rs:50]: i think we have to name it plant_note since there should also be a note for seeds in the future.

Comment (danielsteinkogler) [reply backend/src/model/dto/plantings.rs:50]: so a planting will have a plant_note and a seed_note

Comment (markus2330) [reply backend/src/model/dto/plantings.rs:50]: I'll ask Yvonne if the seeds_note shouldn't directly edit the notes in seeds.

Comment (markus2330) [reply backend/src/model/dto/plantings.rs:50]: She also said that the comment of the seeds should directly reflect the note of the inventory.
Of course this mapping could also be done in the backend, the frontend simply writing to seed_notes.

Comment (horenso) [reply backend/src/model/dto/plantings.rs:50]: Ok so what does that mean?

In the database we now have `notes` in `Seeds` and `Plantings`, but only here in `PlantingDto` it should be called `planting_notes` to distinguish it clearly from seed notes? Should I also populate the field `seed_notes`?

Comment (markus2330) [reply backend/src/model/dto/plantings.rs:50]: > Ok so what does that mean?

To fix what @danielsteinkogler wanted should be easy:

```suggestion
    pub planting_note: Option<String>,
```

With this rename, afaics, the PR already implements #1067.

> Should I also populate the field seed_notes?

To also implement the seed_note better in another PR (as it is a bit more tricky to get consistent when several plantings have the same seed_id).

@danielsteinkogler please make sure that the issues for the backend are up-to-date and describe correctly what you want.

Ideally test if this PR does what you want, please approve the PR when you are happy.

**Review by markus2330 — APPROVED** [review]
LGTM

**Comment by horenso** [comment]
About the names quicky before we finish this:

Seeds already has the field `notes`, that's why I also called it `notes` for plants everywhere. I suggest calling it `planting_notes` (plural)  in `PlantingDto`.

**Comment by horenso** [comment]
Alright I think this is ready to merge @markus2330

**Comment by markus2330** [comment]
@horenso thx! Simply set the label "please merge"  once you think everything is ready. In this case I think, however, that the approval of @danielsteinkogler is pending and the build server has not completed yet.

@danielsteinkogler please approve

**Review by danielsteinkogler — APPROVED** [review]

**Pull request merged**